src/styles/styles.js

```diff
@@ -2339,6 +2343,10 @@ const styles = {
         backgroundColor: colors.black,
         flex: 1,
     },
+
+    errorPageContainer: {
+        backgroundColor: themeColors.componentBG,
+    },
```

Comment:
Do we have a generic class for this background color?

Reply:
There's one attached to `emojiPickerContainer`. Hence I added this.